src/Components/Samples/BlazorUnitedApp/Program.cs

```diff
@@ -16,7 +21,7 @@
 // Configure the HTTP request pipeline.
 if (!app.Environment.IsDevelopment())
 {
-    app.UseExceptionHandler("/Error");
+    app.UseExceptionHandler("/Error", newScope: true);
```

Comment:
How often do we expect this new feature to get used? In templates, samples, inside other extension methods, combine with other overloads, etc.?